SonarQube.Bootstrapper/BootstrapperSettings.cs

```diff
@@ -7,30 +7,44 @@
 
 using SonarQube.Common;
 using System;
-using System.Diagnostics;
 using System.IO;
 
 namespace SonarQube.Bootstrapper
 {
     public class BootstrapperSettings : IBootstrapperSettings
     {
-        // Note: these constant values must be kept in sync with the targets files.
-        public const string BuildDirectory_Legacy = "TF_BUILD_BUILDDIRECTORY";
-        public const string BuildDirectory_TFS2015 = "AGENT_BUILDDIRECTORY";
+        // FIX - should also include the sonar-runner.zip. Consider moving to SonarQube.Common.
+        #region Logical Bootstrapper Version values
 
-        public const string PreProcessorExeName = "SonarQube.MSBuild.PreProcessor.exe";
-        public const string PostProcessorExeName = "SonarQube.MSBuild.PostProcessor.exe";
+        // The following constants determine the "logical" api of the bootstrapper
+        // Server-side: specifies what it expects from the server - where to download the bits from, which exes to run.
+        // Client-side: specifies how the analysis working and bin directories are specified.
 
         /// <summary>
```

Comment:
I've tried to collect all of the constants that affect the "logical bootstrapper API" into the BootStrapperSettings class.